packages/ceramic-core/src/doctype/account-link/account-link-doctype-handler.ts

```diff
@@ -0,0 +1,135 @@
+import CID from 'cids'
+import {
+    AnchorProof,
+    AnchorStatus,
+    DocState, DoctypeConstructor,
+    DoctypeHandler,
+    InitOpts,
+    SignatureStatus
+} from "../../doctype"
+import { Context } from "../../context"
+import { validateLink } from "3id-blockchain-utils"
+import { AccountLinkDoctype, AccountLinkParams } from "./account-link-doctype"
+
+const DOCTYPE = 'account-link'
+
+export default class AccountLinkDoctypeHandler implements DoctypeHandler<AccountLinkDoctype> {
```

Comment:
Does the `DoctypeHandler` need to be a class (that needs to be instantiated) or could it just be an object that has the appropriate functions and properties? 
I don't see a reason it needs to be a class, but maybe I'm missing something?